tests/WatcherFactoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Spatie\PhpUnitWatcher\Test;
+
+use PHPUnit\Framework\TestCase;
+use Spatie\PhpUnitWatcher\WatcherFactory;
+
+class WatcherFactorTest extends TestCase
+{
+	public function setUp() {
+		parent::setUp();
+
+		// todo need to create foo and bar folders inside current working dir
```

Comment:
Handle or remove these todo's.

Reply:
Done in ae8875a6fc6fc44858662669ce33adaf28750892